js/jquery.scripts.js

```diff
@@ -5,4 +5,6 @@ $( document ).ready(function() {
 	var result = test.replace(/\:/g,':<hr />');      
 
 	$('.post-title').html(result);
+
+	$('.recap').prepend("<h2>What did you learn?</h2>");
```

Comment:
Might just be me, but this might be a bit of an overkill :smile: 

Also, missing newline at the end of the file ^^